From f546bdca2585422f4aeb9532891cc7f65dc68306 Mon Sep 17 00:00:00 2001
From: jaderiverstokes <jaderiverstokes@gmail.com>
Date: Wed, 23 Apr 2025 11:03:36 -0700
Subject: [PATCH] patch version

---
 src/JBERC20.sol | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/JBERC20.sol b/src/JBERC20.sol
index 21fb57df..856784e0 100644
--- a/src/JBERC20.sol
+++ b/src/JBERC20.sol
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.23;
+pragma solidity ^0.8.23;
 
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-- 
2.39.5 (Apple Git-154)

